azure-core/azure/core/pipeline/policies/retry.py

```diff
@@ -102,7 +100,8 @@ def configure_retries(self, options):
             'history': []
         }
 
-    def get_backoff_time(self, settings):
+    @staticmethod
+    def get_backoff_time(settings):
```

Comment:
This comment applies to all the changes to make these static methods.
If you're going to make this change you need to change how the methods are called.
For example:
```python
def get_backoff_time(self, settings):
    pass

# This can be called from self
self.get_backoff_time(...)

@staticmethod
def get_backoff_time(settings):
    pass

# This must be called like this:
RetryPolicy.get_backoff_time(...)
```
Also be aware that changing the methods here may have implications for the inherited class AsyncRetryPolicy

Reply:
I would avoid using static methods for this since that would make it much more difficult to extend the retry policy.